internal/bzlmod/go_deps.bzl

```
@@ -689,7 +694,10 @@ def _get_sum_from_module(path, module, sums):
             # replacement have no sums, so we can skip this
             return None
         elif module.local_path == None:
-            fail("No sum for {}@{} from {} found. You may need to run: bazel run @rules_go//go -- mod tidy".format(path, module.raw_version, "parent-label-todo"))  #module.parent_label))
+            # When updating a dependency, its sum may not be in go.sum and we can't hard fail here
+            # since we need Bazel to tidy the module
+            print("No sum for {}@{} found, run bazel run @rules_go//go -- mod tidy to generate it".format(path, module.raw_version))  #module.parent_label))
```

```suggestion
            print("No sum for {}@{} found, run bazel run @rules_go//go -- mod tidy to generate it".format(path, module.raw_version))
```

Let's remove this garbled TODO. @stefanpenner

---

Done